test/unit/gpu/Traversal.cxx

```diff
@@ -28,6 +28,229 @@ using namespace std;
 unsigned s_ntests_run = 0;
 unsigned s_ntests_passed = 0;
 
+template <typename Policy>
+void runSimpleForallTest(Real_ptr test_array,
```

Comment:
Rearranged all of the tests to be in templated functions to simplify policy-switching. None of the actual logic was changed in this file.